esignet-integration-api/src/main/java/io/mosip/esignet/api/validator/Format.java

```diff
@@ -0,0 +1,25 @@
+package io.mosip.esignet.api.validator;
+
+import io.mosip.esignet.api.util.ErrorConstants;
+
+import javax.validation.Constraint;
+import javax.validation.Payload;
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.ElementType.TYPE_USE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+@Target({FIELD, TYPE_USE})
+@Retention(RUNTIME)
+@Constraint(validatedBy = FormatValidator.class)
+@Documented
+public @interface Format {
+    String message() default ErrorConstants.INVALID_CHALLENGE_FORMAT;
```

Comment:
Error message should also convey authFactorType and format mapping incorrectness.